src/index.js

```diff
@@ -11,6 +11,8 @@ import rootSaga from './Root/rootSaga';
 import configureStore from './store/configureStore';
 import Root from './Root/Root';
 
+import '../node_modules/grommet/grommet.min.css';
```

Comment:
Does it mean.. ?

I can't use native import on `.css` file

or 

I need to use `.scss`